proto/types.proto

```diff
@@ -16,11 +16,83 @@ message ExtraData {
 
 // Message to specify network config for a Firecracker VM
 message FirecrackerNetworkInterface {
+	bool AllowMMDS = 1; // Specifies if metadata service should be available on this network interface
```

Comment:
So, uh, generally renumbering fields isn't good here as it's backwards incompatible.  But so is removing fields, and we're doing that because this is a backwards-incompatible change anyway.

I don't think you need to do anything here, but I'm calling this out because it should be abnormal.